doc/requirements.txt

```diff
@@ -0,0 +1,2 @@
+openfisca_core >= 4.2.0, < 5.0
```

Comment:
As I mentioned in the original PR, pip is really bad at managing dependencies with a relative path inside a `requirements.txt` file.

If I put `..` instead of `openfisca_core >= 4.2.0, < 5.0`, it will look for a `setup.py` in the parent directory relatively to _where the instruction is executed from_, and not _where the requirements file is_.